pytest.ini

```diff
@@ -1,3 +1,9 @@
 [pytest]
 filterwarnings =
-    ignore: Converting `np.character` to a dtype is deprecated. :DeprecationWarning
\ No newline at end of file
+    ignore: Converting `np.character` to a dtype is deprecated. :DeprecationWarning
+    # a BIG part of 60k something warnings on pytest -svv fury/tests/test_actors.py are due to deprecated use of <np.data_type> in dipy & vtk, 
+    # where simply data_type could be used, hence ignoring the whole packages.
+    # need to refactor to something where only neglect the np.data_type deprecation warnings
+    once:::dipy
+    once:::vtk
+    
```

Comment:
could you also manage / add this warning: 
```python
 /Users/koudoro/miniconda3/envs/fury-env-37/lib/python3.7/site-packages/vtkmodules/util/numpy_support.py:75: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
  Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
    _vtk_np = {vtkConstants.VTK_BIT:numpy.bool,
```

Reply:
Done!
All we can do is ignore it until the devs for vtkmodules fix it by changing `np.bool` to either `bool` or `np.bool_`

We can get rid of it by changing `once` to `ignore` for those specific packages. i don't want to do that because some warnings are useful (speculating, I have not used this library enough yet :) )

Reply:
I agree, thanks